CHANGELOG.md

```diff
@@ -1,3 +1,46 @@
+Version 1.15.4
+==============
+**January 16, 2023**
+
+This release contains a small number of enhancements and important bug fixes.
+Many thanks to our new contributors @mmorys, @jj-github-jj, and @sandhujasmine, but also our returning contributors @droumis, @jlstevens, @MarcSkovMadsen, @maximlt, @philippjfr, @stanwest, and @Hoxbro.
+
+
+Enhancements:
+- Make lasso select mask values using a Dask-compatible method ([#5568](https://github.com/holoviz/holoviews/pull/5568))
+- Make plotly legend group unique ([#5570](https://github.com/holoviz/holoviews/pull/5570))
+- Set pan and wheel_zoom as the default Bokeh active tools ([#5480](https://github.com/holoviz/holoviews/pull/5480))
+- Enable rendering colorbars on bokeh GraphPlots ([#5585](https://github.com/holoviz/holoviews/pull/5585))
+- Add Plotly Scatter3d documentation and fix colorbar title ([#5418](https://github.com/holoviz/holoviews/pull/5418))
+
+Bug fixes:
+- Only trigger range-update once in callbacks ([#5558](https://github.com/holoviz/holoviews/pull/5558))
+- Ensure dynamically created subplots can be updated ([#5555](https://github.com/holoviz/holoviews/pull/5555))
+- Fix start of stack-level in deprecations ([#5569](https://github.com/holoviz/holoviews/pull/5569))
+- When sorting colormap records, replace None with an empty string ([#5539](https://github.com/holoviz/holoviews/pull/5539))
+- Add deserialization of non-deserialized base64 data ([#5587](https://github.com/holoviz/holoviews/pull/5587))
```

Comment:
Though technically correct, maybe the message could say what is fixed from a user perspective? (e.g. fixes annotator support in geoviews).